crates/api_crud/src/user/create.rs

```diff
@@ -127,7 +127,7 @@ impl PerformCrud for Register {
           "user_already_exists"
         };
 
-        if err_type != "email_already_exists" {
+        if err_type != "email_already_exists" && err_type != "user_already_exists" {
           // If the local user creation errored, then delete that person
           // If we don't exclude 'email_already_exists' we get weird behavior where
           // the existing account is deleted
```

Comment:
Do we need to update this comment to include why `"user_already_exists"` should also be excluded?

Reply:
In my opinion the INSERT operation in Person::create should not have the ON CONFLICT DO UPDATE clause so that need to delete the person in case of errors is eliminated entirely